app/poros/business.rb

```diff
@@ -0,0 +1,12 @@
+class Business
+  attr_reader :name, :address, :business_type, :thumbnail, :yelp_link
+
+  def initialize(result)
+    @name = result[:name]
```

Comment:
Nice poros! Not sure what is convention, but I didn't do a poros file for cocktail details or search. Since we are not using this in the view, I figured I could do with one less file, but maybe our boss Brian 👨🏻‍🏫 will have a different opinion... I don't have a preference but it'll be interesting to know what happens in the wild 🐆 vs school 📚

Reply:
I don't think it's entirely necessary to use poros or even facades with this. But it's good practice. Generally facades are for views rendered for views in a rails app.

Reply:
I was attempting to follow the setup that was in the repo already, where there is a cocktail_facade and a cocktail_details facade. Are these files not being utilized? Can certainly refactor if you would prefer!

Reply:
I don't really care atm, just as long as it does the thing. Can we refactor it later. Just as long as you know why facades are used with a regular rails app versus an api rails app

Reply:
a poro can be absolutely fine to use to prepare the data and send it to the client to free up or clean up a controller though. Facades are a bit more heavy lifting imo for this project

Reply:
@markcyen could you comment on the other files I referenced

Reply:
Honestly, I think a PORO makes a lot of sense here to have things as modular and SRP as possible. I understand the counter arguments, but I think in this case, the business objects ARE different from any of our other endpoints needs... in all the other cases, I think handrolling the serialization makes sense, but here, I actually really like this approach.

Reply:
Maybe a poro will make our boss Brian happy, but I'd be curious what actually happens in the real world 🌎

Reply:
Let's just go with what we have here and then refactor. I think I left the cocktails_details in the poros folder there in case we needed it, but it is currently not being used.